.github/CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @sensu/success
+
+# Order is important. The last matching pattern has the most precedence.
+# So if a pull request only touches javascript files, only these owners
+# will be requested to review.
+*.js    @sensu/services
+
+# Sensu Plugins documentation.
+
+/content/plugins/*      @sensu/plugin-docs
```

Comment:
@majormoses does this line in combination with GitHub team repository write permissions address your concerns?

Reply:
I think this makes a lot of sense.